githubbot/githubbot/handler.go

```diff
@@ -339,15 +340,15 @@ func (h *Handler) handleSubscribeToBranch(repo, branch string, msg chat1.MsgSumm
 			return fmt.Errorf("error creating branch subscription: %s", err)
 		}
 
-		h.ChatEcho(msg.ConvID, "Now subscribed to commits on `%s/%s`.", repo, branch)
+		h.ChatEcho(msg.ConvID, "Now subscribed to notifications for `%s/%s`.", repo, branch)
 		return nil
 	}
 	err = h.db.UnwatchBranch(msg.ConvID, repo, branch)
 	if err != nil {
 		return fmt.Errorf("error deleting branch subscription: %s", err)
 	}
 
-	h.ChatEcho("Okay, you won't receive notifications for commits in `%s/%s`.", repo, branch)
+	h.ChatEcho(msg.ConvID, "Okay, you won't receive notifications for `%s/%s`.", repo, branch)
```

Comment:
how did this even compile before?

Reply:
I think it treated the message string as the ConvID

Reply:
Yeah, but the point of typing with `chat1.ConvIDStr` was to avoid this confusion/mistake

Reply:
I think because it's a string literal it isn't typed as a string at compile time or something like that 😞